client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -187,16 +187,12 @@ export const AppSchemaForm = ({ app }) => {
     initialValues.allocation = app.scheduler;
   }
 
-  // local state for alerts
-  const [visible, setVisible] = useState(true);
-  const onDismiss = () => setVisible(false);
-
   return (
     <div id="appForm-wrapper">
       {jobSubmission.response && (
         <div id="appForm-alerts">
           {jobSubmission.error ? (
-            <Alert color="warning" isOpen={visible} toggle={onDismiss}>
+            <Message type="warn" className="appDetail-error">
```

Comment:
Avoid quiet warnings by (A) using the type "warning", not "warn" (deprecated) and (B) using `SectionMessage`.

```suggestion
            <SectionMessage type="warning" className="appDetail-error">
```